### DISCUSSION TITLE
How to order tree with separate lambdas

### DISCUSSION THREAD

**Discussion by timeforthemoonnight** [hahnlab/CAFE5#174] [Q&A] [answered]
Hi, i would like to run with separate lambda for some branches, but i keep getting erros. I dont know if would actually be possible, but this is my tree
![clade_results](https://github.com/hahnlab/CAFE5/assets/165935538/94664111-f7c1-4280-8158-e22b588d7ec8)
the branches i wanted to separate are (b_supcupularis+b_fungosa)(s_album+s_yasi)(c_campestris+c_australis)(p_japonicum+s_asiatica)

if is possible, how would i write the separeted tree values ?

**Comment by hahnlab-user** [comment]
Can you please paste your Newick-formatted tree here? Then it would be easier to show you the format to use

**Comment by timeforthemoonnight** [comment]
[SpeciesTree_rooted_ultra.txt](https://github.com/hahnlab/CAFE5/files/14938541/SpeciesTree_rooted_ultra.txt)

**Reply by hahnlab-user** [reply, marked as answer]
This should specify different lambdas for each of those four different sets (and one parameter for the background):

((B_subcupularis_representative:2,B_fungosa_representative:2)0.960634:1,(((Bvulgaris_protein:1,Athaliana_protein_:1)0.240857:1,((Slycopersicum_protein:1,(I_cairica_representative:1,(C_campestris_representative:3,C_australis_representative:3)0.981132:1)0.868157:1)0.682739:1,(MguttatusTOL_protein:1,(Lphilippensis_protein:1,(S_asiatica_representative:4,P_japonicum_representative:4)0.332867:1)0.562078:1)0.865595:1)0.639413:1)0.386676:1,(M_oleirifera_representative:1,(S_yasi_representative:5,S_album_representative:5)0.983229:1)0.772187:1)0.960634:1);
